### PR TITLE
chore: Add `README.md`

### PR DESCRIPTION
**Pull request by vrongmeal**
Resolves: #72

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,116 @@
...
+brew install s2-streamstore/s2/s2
+```
+
+### Using Cargo
+
+This method works on any system with [Rust](https://www.rust-lang.org/)
+and [Cargo](https://doc.rust-lang.org/cargo/) installed.
+
+```bash
+cargo install streamstore-cli
```
Comment (infiniteregrets) [inline README.md:74]: need to alias it to s2

Comment (vrongmeal) [reply README.md:74]: I don't think so. Since `[[bin]]` target is called `s2`, the binary will be installed as so.

Comment (infiniteregrets) [reply README.md:74]: oh :p i completely forgot!

Comment (infiniteregrets) [reply README.md:74]: thanks for reminding

**Review by infiniteregrets — APPROVED** [review]
this is awesome!

**Pull request merged**